### ISSUE TITLE
Uncaught Error: Cannot find module 'roaster'

### ISSUE THREAD

**Issue by huglester** [mark-hahn/markdown-scroll-sync#14]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.187.0
**System**: linux 3.16.0-34-generic
**Thrown From**: [markdown-scroll-sync](https://github.com/mark-hahn/markdown-scroll-sync) package, v0.3.2

### Stack Trace

Uncaught Error: Cannot find module 'roaster'

```
At module.js:353

Error: Cannot find module 'roaster'
  at Module._resolveFilename (module.js:351:15)
  at Function.Module._resolveFilename (/usr/share/atom/resources/app/src/module-cache.js:383:52)
  at Function.Module._load (module.js:293:25)
  at Module.require (module.js:380:17)
  at require (module.js:399:17)
  at /home/huglester/.atom/packages/markdown-scroll-sync/lib/main.coffee:13:22
  at process._tickCallback (node.js:366:11)

```

### Commands

```
```

### Config

```json
{}
```

### Installed Packages

```coffee
# User
linter, v0.12.0
linter-php, v0.0.12
linter-phpcs, v0.0.12
linter-phpmd, v0.0.10
markdown-scroll-sync, v0.3.2

# Dev
No dev packages
```

**Comment by mark-hahn** [comment]
thanks for the report.  It is fixed in 0.3.3.

**Comment by robsbx** [comment]
Just installed the package and getting similar error

[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.209.0
**System**: Microsoft Windows 7 Enterprise 
**Thrown From**: [markdown-scroll-sync](https://github.com/mark-hahn/markdown-scroll-sync) package, v0.4.0


### Stack Trace

Uncaught Error: Cannot find module 'roaster'

```
At module.js:330

Error: Cannot find module 'roaster'
    at Module._resolveFilename (module.js:328:15)
    at Function.Module._resolveFilename (C:\Users\michalr1\AppData\Local\atom\app-0.209.0\resources\app.asar\src\module-cache.js:383:52)
    at Function.Module._load (module.js:270:25)
    at Module.require (module.js:357:17)
    at require (module.js:376:17)
    at /C:/Users/michalr1/.atom/packages/markdown-scroll-sync/lib/main.coffee:19:27
    at process._tickCallback (node.js:357:13)
```

### Commands

```
```

### Config

```json
{
  "core": {
    "themes": [
      "one-light-ui",
      "one-dark-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
markdown-scroll-sync, v0.4.0

# Dev
No dev packages
```
